### PR TITLE
[release][step2] Update docs

### PR DESCRIPTION
**Pull request by benoitgrelard**
**This PR is to be merged after #84**

- Adds new `selector` props in all relevant packages
- Add new keyboard shortcut to `DropdownMenu`
- Fix relative path in notes
- Add note to older versions of packages
image
- Remove live previews from older versions of packages
  - @colmtuite this is because we have realised that otherwise we would have to make sure the examples on each versions are running this version's primitive code, which is very difficult technically. So we have opted for leaving the old version's code examples, but just not live preview on them.
image
- Update primitives to latest

### CODE REVIEW

File: pages/primitives/docs/components/dropdown-menu/0.0.2/index.mdx [hunk 1]
```diff
@@ -1192,7 +1335,14 @@ Adheres to the [Menu Button WAI-ARIA design pattern](https://www.w3.org/TR/wai-a
...
     {
       keys: ['Enter'],
-      description: 'Activates the focused item.',
+      description: (
+        <span>
+          When focus is on <Code>DropdownMenu.Trigger</Code>, opens
+          the dropdown menu.
+          <br />
+          When focus is on an item, activates the focused item.
+        </span>
```
Comment (jjenzz) [inline pages/primitives/docs/components/dropdown-menu/0.0.2/index.mdx:1344]: 👍

**Review by jjenzz — APPROVED** [review]

**Comment by peduarte** [comment]
All good for me, just not sure about the note.
- Can we update the copy so we dont have a widow?
- Should we include a link to go to the latest version?

**Review by peduarte — APPROVED** [review]
All good for me, just not sure about the note.

Can we update the copy so we dont have a widow?
Should we include a link to go to the latest version?

**Comment by colmtuite** [comment]
LGTM. Agree with @peduarte, wouldn't be able to sleep well knowing about that widow 😄

**Review by colmtuite — APPROVED** [review]

**Comment by benoitgrelard** [comment]
Silly question, but… what is a widow in this context? 😀

Yeah could add a link to latest, I did think about it. I'll give it a go.

**Comment by colmtuite** [comment]
> what is a widow in this context?

A single word on its own on a new line.

**Comment by benoitgrelard** [comment]
oooooooh!!!! I see TIL. Yes this has been bothering me so much too hahhahaha. I tried different copy but it was always longer that one line… And then I also thought, well, on smaller screens it's gonna wrap anyway… But yeah I agree it would be nice to have one line, or at least a longer second line.

I'll see what it looks like with the link to latest verison.

**Comment by benoitgrelard** [comment]
How is this for you guys?

image

**Comment by benoitgrelard** [comment]
Just reposting this in case you don't see it because GitHub collapsed all the comments:

> How is this for you guys?
image

**Comment by peduarte** [comment]
@benoitgrelard Nice, I think thats better 👌

Just exploring a shorter, more direct route:
image

**Comment by benoitgrelard** [comment]
Yeah I thought about something like that, but it feels a bit harsh to say that it's "outdated" when they might be looking at the version literally just before latest… haha

**Comment by peduarte** [comment]
@benoitgrelard how about something along the lines of
image

**Comment by benoitgrelard** [comment]
I like that a lot. I'll do that.

**Comment by peduarte** [comment]
@benoitgrelard nice one. Im happy to go with this and we can iterate on it if needed.

**Pull request merged**